.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: node_js
 node_js:
-  - "9"
+  - "stable"
```

Comment:
Do you know why bumping to the last stable one would make this work?

Reply:
One of the modules in node_modules has a check to make sure the version of node being run is incompatible with node 9.

`error sane@4.0.3: The engine "node" is incompatible with this module. Expected version "6.* || 8.* || >= 10.*".`

I assume because this is they want you to be using a stable version, or one that is LTS. odd number releases are not LTS.